### DISCUSSION TITLE
Query button will not execute data

### DISCUSSION THREAD

**Discussion by tbailey1225** [pawelsalawa/sqlitestudio#4508] [Q&A]
I haven't had any problems until yesterday, 9/7/22.  After entering the SQL, I click "query" and absolutely nothing populates. The status bar is emply as well. Even if I purposely put in the wrong code, noting populates, not even an error message in the status bar. 

I have restarted my laptop, removed the database, removed SQLite Studio from laptop, redownloaded SQLite Studio (3 times), redownload the database (2 times), I've checked for updates. When I access the history, it shows queries from last week, not any that I've been trying to query in the last 24 hours. It's as if the 'query' button is not working. 

I hope to get this fixed sooner than later, as I'm using this for educational purposes. Picture attached was taken after clicking "query". 
[IMG_2437.pdf](https://github.com/pawelsalawa/sqlitestudio/files/9526674/IMG_2437.pdf)

**Comment by pawelsalawa** [comment]
You are not supposed to click "Query" as this is just a tab header (just like the "History" tab next to it). They are intended to switch between different tabs/views of the window.

To execute query, you should click this:
![image](https://user-images.githubusercontent.com/481846/189138169-fc135c32-70d9-4e6c-b096-eb38486e805e.png)

**Reply by cjuste1** [reply]
Thank you

**Comment by tbailey1225** [comment]
Thank you!

On Thu, Sep 8, 2022 at 8:44 AM Paweł Salawa ***@***.***>
wrote:

> You are not supposed to click "Query" as this is just a tab header (just
> like the "History" tab next to it). They are intended to switch between
> different tabs/views of the window.
>
> To execute query, you should click this:
> [image: image]
> <https://user-images.githubusercontent.com/481846/189138169-fc135c32-70d9-4e6c-b096-eb38486e805e.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pawelsalawa/sqlitestudio/discussions/4508#discussioncomment-3595017>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2SPC6Y3JNHQMMMQEF4WJTDV5HUUTANCNFSM6AAAAAAQHYQGJI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Best,

Tiffany Bailey
